Symfony/CS/Fixer/PSR2/UnneededControlParenthesesFixer.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+/*
+ * This file is part of the PHP CS utility.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Symfony\CS\Fixer\PSR2;
+
+use Symfony\CS\AbstractFixer;
+use Symfony\CS\Tokenizer\Tokens;
+
+// PHP <5.5 support
+if (!defined('T_YIELD')) {
```

Comment:
Yes, but I can't dynamically define a static value.

What is your proposition? :-)